M/MSTM/build_tarballs.jl

```diff
@@ -0,0 +1,35 @@
+using BinaryBuilder, Pkg
+
+name = "MSTM"
+version = v"4.0.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/dmckwski/MSTM.git", "b15798c1ef216cd5e6b6022463dd972b058d372d")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/MSTM/code
+mpifort -O2 mpidefs-parallel.f90 mstm-intrinsics.f90 mstm-v4.0.f90 -o mstm
+cp mstm ${WORKSPACE}/destdir/bin/
```

Comment:
You can just create the file in `${bindir}` instead of creating in the current directory and then moving it :wink: 
```suggestion
mpifort -O2 mpidefs-parallel.f90 mstm-intrinsics.f90 mstm-v4.0.f90 -o "${bindir}/mstm${exeext}"
```